rb/spec/integration/selenium/webdriver/manager_spec.rb

```diff
@@ -64,6 +64,19 @@ module WebDriver
           expect(cookies.first[:value]).to eq('bar')
         end
 
+        it 'should add sameSite cookie with attribute Strict', only: {browser: :chrome} do
+          driver.navigate.to url_for('xhtmlTest.html')
+          driver.manage.add_cookie name: 'foo', value: 'bar', sameSite: 'Strict'
+
+          expect(driver.manage.cookie_named('foo')[:sameSite]).to eq('Strict')
+        end
+
+        it 'should add sameSite cookie with attribute Lax', only: {browser: :chrome} do
+          driver.navigate.to url_for('xhtmlTest.html')
+          driver.manage.add_cookie name: 'foo', value: 'bar', sameSite: 'Lax'
+          expect(driver.manage.cookie_named('foo')[:sameSite]).to eq('Lax')
```

Comment:
Likewise, I believe we need to use snake case 
```suggestion
          expect(driver.manage.cookie_named('foo')[:same_site]).to eq('Lax')
```

Reply:
Updated!